software/chipwhisperer/capture/scopes/OpenADC.py

```diff
@@ -322,39 +325,22 @@ def default_setup(self, verbose=True):
          *  4*7.37MHz ADC clock
          *  tio1 = serial rx
          *  tio2 = serial tx
+         *  tio4 = highZ
          *  CDC settings change off
 
-        Args:
-            verbose (bool): shows which scope settings have changed. Ignores scope.XADC changes,
-                except for scope.XADC.status.
-
         .. versionadded:: 5.1
             Added default setup for OpenADC
         """
         if verbose:
             scope_dict_pre = self._dict_repr()
-        self.gain.db = 25
-        self.adc.samples = 5000
-        self.adc.offset = 0
-        self.adc.basic_mode = "rising_edge"
-        self.clock.clkgen_freq = 7.37e6
-        self.trigger.triggers = "tio4"
-        self.io.tio1 = "serial_rx"
-        self.io.tio2 = "serial_tx"
-        self.io.hs2 = "clkgen"
 
+        self._default_setup()
         self.io.cdc_settings = 0
 
         count = 0
         if self._is_husky:
-            self.clock.clkgen_src = 'system'
-            self.clock.clkgen_freq = 7.37e6
-            self.clock.adc_mul = 4
-            while not self.clock.clkgen_locked:
-                count += 1
-                self.clock.reset_dcms()
-                if count > 10:
-                    raise OSError("Could not lock PLL. Try rerunning this function or calling scope.pll.reset(): {}".format(self))
+            if not self.clock.try_wait_clkgen_locked(10):
```

Comment:
This needs a Husky version as well (`cwhardware/ChipWhispererHuskyClock.py`)

Reply:
This is common logic so for now, in order to not duplicate/copy-paste code, I just moved it to be an OpenADC method with a note to move it into a base/common clock class if refactoring ever happens.